controller/array_action/array_mediator_ds8k.py

```diff
@@ -447,7 +447,7 @@ def get_snapshot(self, snapshot_name):
             raise array_errors.SnapshotNameBelongsToVolumeError(target_api_volume.name,
                                                                 self.service_address)
         fcrel = self._get_flashcopy(target_api_volume.flashcopy[0].id)
-        source_volume_name = self.get_volume_name(fcrel.source_volume.id)
+        source_volume_name = self.get_volume_name(fcrel.source_volume['id'])
```

Comment:
recursively `Munch` fcrel (now `flashcopy_process`) in `ds8k_rest_client.py`, to avoid this change